packages/jmix-front-generator/src/generators/react-typescript/entity-management-graphql-hooks/index.ts

```diff
@@ -0,0 +1,37 @@
+import {ComponentOptions, componentOptionsConfig} from "../../../common/cli-options";
+import * as path from "path";
+import {defaultPipeline} from "../../../building-blocks/pipelines/defaultPipeline";
+import {allQuestions, Answers, getAnswersFromPrompt} from "./answers";
+import { Options } from "./options";
+import {TemplateModel, deriveTemplateModel} from "./template-model";
+import {write} from "./write";
+import {YeomanGenerator} from "../../../building-blocks/YeomanGenerator";
+
+export class React_EntityManagement_GraphQL_Hooks extends YeomanGenerator {
+
+  constructor(args: string | string[], options: ComponentOptions) {
+    super(args, options);
+  }
+
+  async generate() {
+    await defaultPipeline<Options, Answers, TemplateModel>({
+      templateDir: path.join(__dirname, 'template'),
+      questions: allQuestions, // Used when refining answers
+      stages: { // Using custom implementations for some of the stages
+        getAnswersFromPrompt,
```

Comment:
As for me, it will be more clear, if we explicitly listed all stages here. A bit more lines of code but less magic. @web-devel any thoughts?

Reply:
@vadimbasko current version is ok to me, imo otherwise there will be to much boilerplate